iso/preseeds/preseed_unattended.cfg

```diff
@@ -61,7 +62,7 @@ d-i mirror/country string manual
 d-i mirror/http/hostname string deb.debian.org
 d-i mirror/http/directory string /debian
 d-i mirror/http/proxy string
-d-i apt-setup/local0/repository string http://deb.debian.org/debian/ bullseye main contrib non-free
+d-i apt-setup/local0/repository string http://deb.debian.org/debian/ bookworm main contrib non-free
```

Comment:
Same comment as before.

Reply:
same comment as above for me too. I will continue to look stuff over make tweaks and test again with my node.  
If anyone else in @dappnode/core has suggestions please  let me know and i can try it out!.